src/plugin/regions.js

```diff
@@ -45,6 +45,11 @@ class Region {
         this.scroll = params.scroll !== false && ws.params.scrollParent;
         this.scrollSpeed = params.scrollSpeed || 1;
         this.scrollThreshold = params.scrollThreshold || 10;
+        // Determines whether the context menu is prevented from being opened.
+        this.preventContextMenu =
```

Comment:
the `preventContextMenu` needs to be documented (see other params).

Reply:
I'm not sure what you mean. The "documentation" is in the comment on line 48. Do I need to document it (also?) somewhere or do you mean that the comment needs to be improved?

Reply:
see params documentation at https://github.com/katspaugh/wavesurfer.js/blob/2943113705caaae1ea9c23fd9516ac1c1d0c9b61/src/plugin/regions.js#L595

Reply:
Done. Also included the parameter `handleStyle` (PR #1798), which was also missing from the docs.